### DISCUSSION TITLE
AsciiDoc / adoc preview does not work

### DISCUSSION THREAD

**Discussion by fabiovinci** [community/community#55286] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

It seems GitHub is no more able to generate previews for some AsciiDoc / adoc files.

After the selection of the file (e.g. https://github.com/INSPIRE-MIF/technical-guidelines/blob/main/data/so/dataspecification_so.adoc) GitHub shows the asciidoc code instead of the preview.

![image](https://github.com/community/community/assets/22212406/0096e5b6-c4d6-48b4-8412-7569aa009316)

In the editing mode, the preview shows an error message. 

![image](https://github.com/community/community/assets/22212406/b5cfc9cb-2af3-4de6-bf02-83eb384310bc)

Do you have an idea about the source of the error?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by kgrzybek** [comment]
We have the same issue. 

I think it is somehow related with the size of the file - for smaller files preview works, for bigger ones - it does not.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @fabiovinci ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by fabiovinci** [comment]
Dear all,

it seems the issue with the preview is no longer present, at least for the AsciiDoc files available in the INSPIRE repository.

Do you have any news about that?
Has a fix been implemented?
